resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -309,6 +308,11 @@
 close SomaticFile;
 close Ann;
 
+### Crashing when there is less than 50 germline variants
+if($json{'germlineSmallVarsInBothRare'} < 50){
+  die "Less than 50 rare germline variants. Might be a sample swap or poor coverage on any one of the samples, please check the coverage information.\nExiting the analysis";
+}
+
 ## if only TiNDA needs to run with annotation files already available
 TiNDA:
```

Comment:
Hooooh! I see this is a "goto" target! "Goto" is really a no-go at last since 1980. Better make if-else branches, optionally put the code in the branches into functions, if you need to separate the code.

Reply:
Sure, this GOTO could go out of here, will update in a different pull request.